PolyEngine/Engine/Src/TransformComponent.hpp

```diff
@@ -0,0 +1,34 @@
+#pragma once
+
+#include "ComponentBase.hpp"
+
+namespace Poly {
+	class TransformComponent : public ComponentBase
+	{
+	public:
+		TransformComponent() {};
```

Comment:
This is unnecessary. Default constructor will be created automatically. Delete this line.